frontend/public/components/utils/dropdown.jsx

```diff
@@ -353,18 +357,20 @@ export class Dropdown extends DropdownMixin {
 Dropdown.propTypes = {
```

Comment:
Just reordered these in alphabetical order and added the `headerBefore` and `spacerBefore` props.